File: 40130112056/Rotate-Array.py
def rotate_array(arr, k):
    n = len(arr)
    k = k % n
    rotated_arr = [0] * n
    for i in range(k):
        rotated_arr[i] = arr[n-k+i]

    for i in range(k, n):
        rotated_arr[i] = arr[i-k]
    
    return rotated_arr

array = [1, 2, 3, 4, 5]
k = 2
print(rotate_array(array, k))

      

      
    

File: 40130212009/rotateArray.js
function rotateArray(arr, k) {
  let n = arr.length;
  k = k % n;
  let rotatedArr = new Array(n);
  for (let i = 0; i < k; i++) {
    rotatedArr[i] = arr[n - k + i];
  }

  for (let i = k; i < n; i++) {
    rotatedArr[i] = arr[i - k];
  }

  return rotatedArr;
}

let array = [1, 2, 3, 4, 5];
let k = 2;
console.log(rotateArray(array, k));
